index.html

```diff
@@ -3770,7 +3769,6 @@ <h5>JWT Decoding</h5>
   "nbf": 1541493724,
   "iat": 1541493724,
   "exp": 1573029723,
-  "nonce": "343s$FSFDa-",
```

Comment:
This seems like a mistake... presentations are supposed to support replay attack mitigation. by removing the nonce from this example, we are muddying those waters.

Reply:
Does it make sense to move this in't the vp section as you mentioned on the call @David-Chadwick